src/components/List_Products.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
+import { saveItem } from '../lib/api';
+import { db } from '../lib/firebase';
+import { collection, onSnapshot } from 'firebase/firestore';
+
+function ListProducts() {
+  const [items, setItems] = useState(null);
```

Comment:
@candrepa1 : I believe that it might be better as an empty array (`[]`). 😄 

```suggestion
  const [items, setItems] = useState([]);
```